src/app/cases/cases-list/cases-list.component.ts

```diff
@@ -105,7 +105,14 @@ export class CasesListComponent implements OnInit, OnDestroy {
       width: CASE_EXPORT_CUSTOM_MODAL_WIDTH,
     });
   }
-
+  setQueryParamsInFilters(): Filter[] {
```

Comment:
we could move this into helper service, so we don't have to duplicate it

Reply:
Done.